baselines/niid_bench/README.md

```diff
@@ -0,0 +1,108 @@
+---
+title: "Federated Learning on Non-IID Data Silos: An Experimental Study"
+url: https://arxiv.org/abs/2102.02079
+labels: [data heterogeneity, image classification, benchmark]
+dataset: [cifar10, mnist, fashion-mnist]
+algorithms: [fedavg, scaffold, fedprox, fednova]
+---
+
+# Federated Learning on Non-IID Data Silos: An Experimental Study
+
+****Paper:**** : [https://arxiv.org/abs/2102.02079](https://arxiv.org/abs/2102.02079)
```

Comment:
```suggestion
**Paper:** [https://arxiv.org/abs/2102.02079](https://arxiv.org/abs/2102.02079)
```

Reply:
note how I changed `****` to `**` and also removed the second `:`. This appears in many places in the readme. Could you please fix them all?